cfgov/paying_for_college/views.py

```diff
@@ -309,6 +311,33 @@ def school_search_api(request):
     return HttpResponse(json_doc, content_type='application/json')
 
 
+def school_autocomplete(request):
+    search_term = request.GET.get('q', '').strip()
+    if not search_term:
+        return HttpResponse(json.dumps([]), content_type='application/json')
+    results = SchoolSearch(search_term).autocomplete()
+
+    return HttpResponse(json.dumps(results), content_type='application/json')
+
+
+def school_search(request):
+    search_term = request.GET.get('q', '').strip()
+    if not search_term:
+        return HttpResponse(json.dumps([]), content_type='application/json')
+    response = SchoolSearch(search_term).search()
+
+    document = [{'schoolname': school.text,
+                 'id': school.school_id,
+                 'city': school.city,
+                 'nicknames': str(school.nicknames),
+                 'state': school.state,
+                 'zip5': school.zip5,
+                 'url': school.url}
+                for school in response.get('results')]
+
+    return HttpResponse(json.dumps(document), content_type='application/json')
```

Comment:
I think school_autocomplete is the only es7 view we need. There is no other use of PFC search.

```suggestion
```

Reply:
Done. I replaced `school_search` with `school_autocomplete` as the only ES7 view.